README.md

```diff
@@ -78,6 +76,7 @@ client := pusher.Client{
 ```go
 client := pusher.ClientFromURL("http://key:secret@api.pusherapp.com/apps/app_id")
```

Comment:
Maybe change this so it's an api url that includes a cluster (ie api-mt1 or whatever)